<launch>
    <arg name="run_robot"   default="false"/>
    <!-- <arg name="run_robot"   default="false"/> -->

    <group if="$(arg run_robot)">
        <node name="pioneer_main" pkg="pioneer_main"  type="main_cross_arm.py"  output="screen"/>
    </group>

    <node name="pioneer_main"     pkg="pioneer_main"  type="cross_arm_l2pc.py"  args="$(arg run_robot)" output="screen"/>
</launch>